apps/coscrad-frontend-e2e/src/support/commands.ts

```diff
@@ -5,6 +5,8 @@ declare namespace Cypress {
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     interface Chainable<Subject> {
         login(email: string, password: string): void;
+
+        dragMapFromCenter(element): void;
```

Comment:
Remove this. This is definitely not the place to be building test helpers like this.